docs/xml-config-file.rst

```diff
@@ -156,3 +164,77 @@ too.
+
+Your *Bridge Library* must define at least a publisher to your desired protocol and it is responsible to
+communicate with it and follow the ISPublisher interface. By default, the transformation function is applied after
+:class:`on_received_data` method calls to the instance of ISBridge.
+If you want to change this behaviour you will need to override the complete data flow.
+
+*Bridge_configuration* node can contain configuration information that *Bridge Library* must understand.
+ISManager will parse the *property* nodes of each element and will call the respective :class:`create_`
+function of the library with a vector of pairs with the data contained.
+If no *bridge_configuration* is provided, then your createBridge will be called with :class:`nullptr` or an empty
+vector as parameter config.
+
+*Transformation* library could be reused by your bridge library, with the same or another transformation function inside the same transformation library (an example of reusing the transformation library can be found on `FIROS2 <https://github.com/eProsima/FIROS2/tree/master/examples/TIS_NGSIv2>`__.
+Of course, you can add built in transformation functions inside your *bridge library*.
+
+.. image:: IS-RTPS-to-Other.png
+    :align: center
+
+Other procotol to RTPS
+^^^^^^^^^^^^^^^^^^^^^^
+
+This is a similar case as the previous one, but in the other way, as in the connector *protocol_shapes* of our example.
+
+The same logic applies in this connectors as in the :ref:`rtps to other protocol` case,
+but in this case the RTPS participant is the publisher. An example of this can be found on
+`FIROS2 <https://github.com/eProsima/FIROS2/tree/master/examples/helloworld_ros2>`__.
+
+.. image:: IS-Other-to-RTPS.png
+    :align: center
+
+Bidirectional bridge
+^^^^^^^^^^^^^^^^^^^^
+
+This case is not a connector, but the consecuence of set two connectors with the correct parameters.
+In our example the combination of *shapes_projection* and *shapes_stereo* is a bidirectional bridge,
+as well as, *shapes_protocol* and *protocol_shapes*.
+
+A combination of both logics :ref:`rtps to other protocol` and :ref:`Other procotol to RTPS` applies here.
+The example `TIS_NGSIv2 <https://github.com/eProsima/FIROS2/tree/master/examples/TIS_NGSIv2>`__ of FIROS2 uses a
+bridge of this type.
+
+.. image:: IS-RTPS-Other.png
+    :align: center
+
+Example
+-------
+
+In this file there are defined two RTPS *participants*, and a *bridge*. All of them have a subscriber and a publisher.
+The relationships between *participants* and *subscribers*/*publishers* defined in the *profiles* section are
+stablished by each *connector*. This allows to share *subscribers*/*publishers* configurations between *participants*.
+There are four connectors defined: *shapes_projection*, *shapes_stereo*, *shapes_protocol* and *protocol_shapes*.
+
+.. figure:: Config.png
+    :align: center
+    :target: example_config.xml
```

Comment:
dont link the xml, i saw it because i read the raw RST. But a user can think it is only an image. Put the xml after the image. I think is more clear.

Reply:
There is a image footer that informs the user about that with "Click on the image to open the code."

Reply:
Added xml too.